HtmlMesh/src/pointer-events-capture-behavior.ts

```diff
@@ -37,73 +32,92 @@ const startCaptureOnEnter = (scene: Scene) => {
     if (captureOnEnterCount === 0) {
         document.addEventListener("pointermove", onPointerMove);
         _scene = _scene ?? scene;
-        let rect = _scene
-            .getEngine()
-            .getRenderingCanvasClientRect() as ClientRect;
-        _canvasRect =
-            _canvasRect ??
-            new DOMRect(rect.left, rect.top, rect.width, rect.height);
+        console.log(
```

Comment:
Can you use Logger.log from the Babylon logger instead ?

Reply:
Done